suites/quincy/rgw/tier-0_rgw.yaml

```diff
@@ -0,0 +1,123 @@
+# Tier 0: RGW build evaluation
```

Comment:
Could we rename the test suite filename that is more appropriate? The below tests look more like functional testing i.e. `Tier-1`